ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -140,16 +140,21 @@ func displayAsTable(containers []types.Container) {
 	w := new(tabwriter.Writer)
 
 	w.Init(os.Stdout, cellWidthInSpaces, widthBetweenCellsInSpaces, cellPaddingInSpaces, paddingCharacter, noFormatting)
-	fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tSTATUS\tPORTS\tNAMES\tTASKDEFINITION\tTASKFILEPATH")
+	fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tSTATUS\tPORTS\tNAMES\tTASKDEFINITION")
 	for _, container := range containers {
-		row := fmt.Sprintf("%s\t%s\t%s\t%s\t%s\t%s\t%s",
+		// The ARN/Family and FilePath labels are mutually exclusive
+		taskDef := container.Labels[taskDefinitionLabelKey]
+		if container.Labels[taskFilePathLabelKey] != "" {
```

Comment:
nit: I thought this was going to be condensed into a single label for both

Reply:
It is :) They appear under the TASKDEFINITION column.

Reply:
Yeah I saw that- I meant the docker labels themselves would be the same. But its alright... this is a minor thing..

Reply:
🤦‍♂️ fixed it, this was nice went from 3 constants to having only 1 :)